ci/test/00_setup_env_native_tidy.sh

```diff
@@ -20,6 +20,7 @@ export RUN_TIDY=true
 export GOAL="install"
 export BITCOIN_CONFIG="\
  -DWITH_ZMQ=ON -DBUILD_GUI=ON -DBUILD_BENCH=ON -DWITH_USDT=ON \
+ -DBUILD_UTIL_CHAINSTATE=ON -DBUILD_KERNEL_LIB=ON -DBUILD_KERNEL_TEST=ON \
```

Comment:
i wonder if this can use the dev-mode preset, so that only a single list has to maintained for all targets

Reply:
sgtm